### DISCUSSION TITLE
[Feature Request] The timing of the Ripples effect

### DISCUSSION THREAD

**Discussion by a644994** [heroui-inc/heroui#4726] [Feature requests]
### Is your feature request related to a problem? Please describe.

Jumping to the page in the click event, unable to see the Ripples effect

### Describe the solution you'd like

Can I modify the Ripples effect to be executed on onMouseUp  

### Describe alternatives you've considered

onMouseUp -> onClick

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1428/[feature-request]-the-timing-of-the-ripples-effect">ENG-1428 [Feature Request] The timing of the Ripples effect</a></p>
